rules.neon

```diff
@@ -3,3 +3,4 @@ rules:
 	- PHPStan\Rules\PHPUnit\AssertSameNullExpectedRule
 	- PHPStan\Rules\PHPUnit\AssertSameWithCountRule
 	- PHPStan\Rules\PHPUnit\MockMethodCallRule
+	- PHPStan\Rules\PHPUnit\ShouldCallParentMethodsRule
```

Comment:
Instead of adding this as a new rule which would block another 0.12.x release, could you:

1) Require PHPStan 0.12.33 as it contains the bleedingEdge parameter
2) Add this as a conditional rule like this:

```
conditionalTags:
    PHPStan\Rules\PHPUnit\ShouldCallParentMethodsRule:
        phpstan.rules.rule: %featureToggles.bleedingEdge%

services:
    -
        class: PHPStan\Rules\PHPUnit\ShouldCallParentMethodsRule
```

Reply:
Once we have it in conditionalTags, it should no longer be in the rules section.